Code Examples/week4/bridge-vue/README.md

```diff
@@ -1232,3 +1232,152 @@ These techniques can be used in your own applications and components.  Also, now
 that you have a better understanding of how to build component based apps, you can
 start exploring using the many third-party Vue components that exist.  Whether you
 write it, or someone else does, the ideas are the same.
+
+
+## Search Bar 
+
+It is an optional augmentation, granted we mind minimal use case for the application.
+![Screencast of SearchBar in Action](screenshots/search-bar.gif)
+
+[Semantic-UI-Vue](https://github.com/Semantic-UI-Vue/Semantic-UI-Vue) is a [Semantic-UI](https://semantic-ui.com/) integration, while Semantic is a flexbox friendly UI framework designed for theming. It will bring about pretty elements for input field, loader and error-modal for `bridge-vue` application.
```

Comment:
```suggestion
In addition to extending our app, we'll also gain some experience working with third-party libraries.  Specifically, we'll use [Semantic-UI-Vue](https://github.com/Semantic-UI-Vue/Semantic-UI-Vue). It will bring about pretty elements for the input field, loader and error-modal.
```